index.js

```diff
@@ -419,6 +419,55 @@
     return f(y, x);
   });
 
+  //# lift :: Apply f => (a -> b) -> f a -> f b
+  //.
+  //. Promotes a unary function to a function which operates on a Functor.
+  //.
+  //. ```javascript
+  //. > S.lift(R.inc, S.Just(2))
+  //. S.Just(3)
+  //.
+  //. > S.lift(R.inc, S.Nothing())
+  //. S.Nothing()
+  //. ```
+  S.lift = def('lift', [Function, a], R.map);
+
+  //# lift2 :: Apply f => (a -> b -> c) -> f a -> f b -> f c
+  //.
+  //. Promotes a binary function to a function which operates on two Apply's.
+  //.
+  //. ```javascript
+  //. > S.lift2(R.add, S.Just(2), S.Just(3))
+  //. S.Just(5)
+  //.
+  //. > S.lift2(R.add, S.Just(2), S.Nothing())
+  //. S.Nothing()
+  //.
+  //. > S.lift2(S.and, S.Just(true), S.Just(true))
+  //. S.Just(true)
+  //.
+  //. > S.lift2(S.and, S.Nothing(), S.Just(false))
+  //. S.Nothing()
```

Comment:
Let's replace this example with `S.lift2(S.and, S.Just(true), S.Just(false))` since one of the preceding examples demonstrates the `Nothing()` behaviour.